.github/workflows/standalone-fleet-ci.yml

```
@@ -70,36 +70,37 @@ jobs:
             --agents 3
             --network "nw01"
       -
-        name: K3d import images
+        name: K3d Import Images
```

nit: consistent naming of `k3d` and format of naming workflows with `<verb>...`
```suggestion
        name: Import Images Into k3d
```
```suggestion
        name: K3d Import Images
```